agent/ecscni/netconfig_windows.go

```diff
@@ -16,13 +16,23 @@
 package ecscni
 
 import (
-	"github.com/containernetworking/cni/pkg/types"
+	"regexp"
 
 	"github.com/aws/amazon-ecs-agent/agent/api/eni"
+
+	"github.com/cihub/seelog"
 	"github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/cni/pkg/types"
 	"github.com/pkg/errors"
 )
 
+const (
+	// maxInputLength is the maximum length of IP Address or MAC Address as input to CNI plugin.
+	maxInputLength = 18
```

Comment:
Is this length per field?

Reply:
That's the maximum length of any field.
IP Address would be 18 characters long and 17 characters for MAC Address.